### ISSUE TITLE
NickJS does not work with recent versions of Chrome (v64+): Network.setRequestInterceptionEnabled wasn't found

### ISSUE THREAD

**Issue by Leftium** [phantombuster/nickjs#23]
**This error occurs after upgrading to latest version of Chrome:**

`error when initializing new chrome tab: Error: 'Network.setRequestInterceptionEnabled' wasn't found`

**Confirmed with these release channels of Chrome:**

- Release (Version 64.0.3282.119 (Official Build) (64-bit))
- Beta (Version 64.0.3282.119 (Official Build) (64-bit))
- Dev (Version 65.0.3325.18 (Official Build) dev (32-bit))

**Environment:**

- Windows 10
- NickJS 0.2.6
- Node v8.9.1

**The code:**
``` coffeescript
Nick = require 'nickjs'
nick = new Nick()

nick.newTab().then (tab) =>
    await console.log "Testing..."
.then () =>
    console.log 'Completed with no errors!'
    nick.exit 0
.catch (err) =>
    console.log 'Something went wrong:'
    console.log err
    nick.exit 1
```

**Comment by Leftium** [comment]
Seems to be related to this: https://github.com/GoogleChrome/puppeteer/pull/1153

> BREAKING CHANGE
> 
> Page.setRequestInterceptionEnabled is renamed into
> Page.setRequestInterception.

**Comment by Leftium** [comment]
I found a work-around by installing a previous version of Chromium from https://chromium.woolyss.com/

(Specifically [this download](https://github.com/henrypp/chromium/releases/download/v63.0.3239.132-r508578-win64/chromium-sync.zip))

Hopefully using Chromium instead of Chrome will also prevent problems caused due to Chrome's auto-update.

**Comment by paps** [comment]
I should be able to patch this tomorrow

**Comment by paps** [comment]
Fixed in [v0.3.0](https://github.com/phantombuster/nickjs/releases/tag/v0.3.0) by 04d5216e593ecac37b65b4cdb60290f77b7b3df0
